src/app/screens/nftDashboard/collectiblesTabs.tsx

```diff
@@ -246,7 +246,7 @@ export default function CollectiblesTabs({
                 text={t('LOAD_MORE')}
                 processing={rareSatsQuery.isFetchingNextPage}
                 disabled={rareSatsQuery.isFetchingNextPage}
-                onPress={rareSatsQuery.fetchNextPage}
+                onPress={() => rareSatsQuery.fetchNextPage()}
```

Comment:
is this needed? 🤔

Reply:
Yeah, Tim just added this as a fix for this error
![image](https://github.com/secretkeylabs/xverse-web-extension/assets/36603049/ca88c9c2-9272-4f8d-aca9-cd5f12bbfd17)

Reply:
yeah type errors: 
![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/60a8cec5-8f41-480d-bc27-ddb2eb15c88e)

https://secretkeylabs.slack.com/archives/C05QCJM9D1N/p1701250509584929